test/test_rdoc_ruby_lex.rb

```diff
@@ -789,28 +789,16 @@ def test_class_tokenize_symbol
     assert_equal expected, tokens
   end
 
-  def test_class_tokenize_particular_kind_of_symbols
```

Comment:
Is it a regression?

`est_class_tokenize_particular_kind_of_symbols` is still working test case.